src/components/Changeset/ChangesetBadge.tsx

```diff
@@ -71,10 +71,18 @@ export interface ChangesetBadgeProps extends UseStyles<typeof useStyles> {
   disableOutline?: boolean;
   moreInfo?: ReactNode;
   TooltipProps?: Omit<TooltipProps, 'title' | 'children'>;
+  anchorHorizontal?: 'right' | 'left';
```

Comment:
I'd prefer to be more consistent with upstream prop
```suggestion
  anchorOrigin?: Partial<BadgeOrigin>;
```
```ts
anchorOrigin={{ horizontal: 'right' }}
```